packages/plugins/plugin-deck/src/components/DeckLayout/Plank.tsx

```diff
@@ -189,7 +191,7 @@ const SplitFrame = ({ children }: PropsWithChildren<{}>) => {
   return (
     <div
       role='none'
-      className={mx('grid grid-cols-[1fr_1fr] absolute inset-0', railGridHorizontal, mainIntrinsicSize)}
+      className={mx('grid grid-cols-[1fr_1fr] absolute inset-0 min-bs-0', railGridHorizontal, mainIntrinsicSize)}
```

Comment:
Did this not do anything? We can remove it if it had no effect, but it'll do no harm to leave in-place.

Reply:
oh right, yeah I forgot about this, will revert